docs/data/base/components/modal/KeepMountedModal.js

```diff
@@ -23,8 +23,12 @@ export default function KeepMountedModal() {
         keepMounted
       >
         <ModalContent sx={style}>
-          <h2 id="keep-mounted-modal-title">Text in a modal</h2>
-          <p id="keep-mounted-modal-description">Aliquid amet deserunt earum!</p>
+          <h3 id="keep-mounted-modal-title" className="modal-title">
```

Comment:
Is it intentional to change the DOM node? From a11y point this should be h2 from what I know (or even h1). I would keep it as is, as it is consistent with Material UI, and we can just update the styles. If we decide that we should use h1 as a title, we can change it in one o everywhere.